kamelet-support/src/test/java/io/kaoto/backend/metadata/parser/step/camelroute/CamelRouteParseCatalogTest.java

```diff
@@ -140,6 +140,17 @@ private void assertBrowseJsonHasBeenParsedCorrectly(
         );
     }
 
+    @Test
+    void checkDuration() {
+        String camelZip = "resource://camel-3.19.0.zip";
```

Comment:
migth be worthy to extract the camelZip as a constant as it is used several times in the class